src/languages/smarty.coffee

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Need to change https://github.com/Glavin001/atom-beautify/blob/master/src/languages/index.coffee#L14 to add `smartly` (`smart` ==> `require("smartly.coffee")`).